### PR TITLE
Image API rights properties

### LINKED ISSUE #1415: Rename attribution to requiredStatement in Image API
See #1287 (closed by #1413 as complete for Presentation API 3), we need to replicate this for Image API 3.

### PR DESCRIPTION
**Pull request by zimeon**
Make Image API `requiredStatement`, `rights` and `logo` follow prezi in parallel with #1504

Closes #1415

### CODE REVIEW

File: source/api/image/3.0/index.md [hunk 1]
```diff
@@ -591,28 +591,18 @@ Objects in the `tiles` list _MUST_ each have a unique combination of `width` and
...
+| `logo` | Optional | A small external image resource that represents an individual or organization associated with this image. This could be the logo of the owning or hosting institution. The value of this property _MUST_ be an array of JSON objects, each of which _MUST_ have an `id` and should have at least one of `type` and `format`. The logo _MUST_ be clearly rendered when the resource is displayed or used, without cropping, rotating or otherwise distorting the image. It is _RECOMMENDED_ that a IIIF Image API [service](#related-services) be available for this image for other manipulations such as resizing.  |
 {: .api-table}
 
-All of the rights and licensing properties _MAY_ have multiple values, expressed as a JSON array, or a single value.
+It is _RECOMMENDED_ that logos with IIIF Image API services do not, themselves, have `logo` properties. When clients render logos specified with an IIIF Image API service, they _MAY_ ignore any `logo` property on in the included logo.
```
[the remark this answers is not in the document]
Comment (zimeon) [reply source/api/image/3.0/index.md]: Aye

File: source/api/image/3.0/index.md [hunk 2]
```diff
@@ -591,28 +591,18 @@ Objects in the `tiles` list _MUST_ each have a unique combination of `width` and
...
-| `logo` | Optional | A small image that represents an individual or organization associated with the content. Logo images _MUST_ be clearly rendered when content obtained from the Image API service is displayed or used. Clients _MUST NOT_ crop, rotate, or otherwise distort the image.   |
+| `requiredStatement` | Optional  | Text that _MUST_ be shown when content obtained from the Image API service is displayed or used. It might include copyright or ownership statements, or a simple acknowledgement of the providing institution. The value of this property _MUST_ be a JSON object, that has the `label` and `value` properties. The values of both `label` and `value` must be JSON objects, as described in the [Language of Property Values][prezi-languages] section of the Presentation API. The `value` property _MAY_ contain simple HTML as described in the [HTML Markup in Property Values][prezi-html] section of the Presentation API. Given the wide variation of potential client user interfaces, it will not always be possible to display this statement to the user in the client’s initial state. If initially hidden, clients _MUST_ make the method of revealing it as obvious as possible. |
```
[the remark this answers is not in the document]
Comment (zimeon) [reply source/api/image/3.0/index.md]: The classic approach to such problems is to pull out the commonality into some other spec that both can rely upon, but that doesn't appeal to me (yet)

File: source/api/image/3.0/index.md [hunk 3]
```diff
@@ -591,28 +591,18 @@ Objects in the `tiles` list _MUST_ each have a unique combination of `width` and
...
+| `rights` | Optional | A string that identifies a license or rights statement that applies to the content of this image. The value of this property _MUST_ be drawn from the set of [Creative Commons][cc-licenses] licenses, the [RightsStatements.org][rs-terms] rights statements, or those added via the [extension][prezi-ldce] mechanism. The inclusion of this property is informative, and for example could be used to display an icon representing the rights assertions. If displaying rights information directly to the user is the desired interaction, or a publisher-defined label is needed, then it is _RECOMMENDED_ to include the information using the `requiredStatement` property. |
```
[the remark this answers is not in the document]
Comment (zimeon) [reply source/api/image/3.0/index.md]: Good suggestion, will fix that

**Comment by azaroth42** [comment]
:+1:  @tomcrane can merge

**Pull request merged**